src/107-Arduino-TMF8801.cpp

```diff
@@ -42,6 +42,9 @@ ArduinoTMF8801::ArduinoTMF8801(TMF8801::I2cWriteFunc write,
 , _calib_data{calib_data}
 , _algo_state{algo_state}
 {
+  /* set gpio_control to Input for GPIO0 and GPIO1
+   */
+  _gpio_control=0x00;
```

Comment:
Thank you for putting the initialisation into the Ctor. If you put it now up a couple of lines (either before or after the existing initialisations of e.g. _calib_data) I shall be most grateful :wink: